### DISCUSSION TITLE
Student Verification is Impossible for Online Learners

### DISCUSSION THREAD

**Discussion by Cupara** [community/community#52015] [GitHub Education]
I'm attending Maryville University of St. Louis online. They provide a Current Enrollment Verification Certificate which I have attached. But I get the error:

- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
My university doesn't provide this in any type of documentation.

- Your image should be one of the following:
- Dated school ID
- Dated official/unofficial transcript
My school doesn't provide dated school IDs. I tried using my unofficial transcript but still was denied.

On my old account, it was easier to get verified as a student but now it is near impossible with this whole automated system and you can't even contact support any longer to get manual verification done.

Moderator Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by samus-aran** [comment]
Hi @Mythotical ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. This discussion will be closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden/locked the duplicate comments/discussions, and we ask that you no longer cross-post.

We'll use your [previous discussion](https://github.com/orgs/community/discussions/51791) to converse and see how we can assist. ⭐
